src/Pos/Wallet/Web/Server/Methods.hs

```diff
@@ -332,7 +332,7 @@ updateUserProfile profile = setProfile profile >> getUserProfile
 
 getWallet :: WalletWebMode ssc m => CAddress -> m CWallet
 getWallet cAddr = do
-    balance <- getBalance =<< decodeCAddressOrFail cAddr
+    balance <- fmap mkCCoin . getBalance =<< decodeCAddressOrFail cAddr
     meta <- getWalletMeta cAddr >>= maybe noWallet pure
     pure $ CWallet cAddr balance meta
   where
```

Comment:
Because client-api (purescipt) was using `Int` before which is bounded by:
```
> import Prelude
> top :: Int
2147483647
```
which is way less then 2^56 that we have to support at least.

So before this javascript side used Number (max 2^53 - 1) or Int (max 2147483647) to hold money and send it. Now they use big-number and string to hold and send valud